src/Fixer/ClassNotation/ClassAttributesSeparationFixer.php

```diff
@@ -39,24 +41,26 @@
  */
 final class ClassAttributesSeparationFixer extends AbstractFixer implements ConfigurableFixerInterface, WhitespacesAwareFixerInterface
 {
-    /**
-     * @internal
-     */
-    public const SPACING_NONE = 'none';
+    private const SPACING_NONE = 'none';
```

Comment:
@keradus @kubawerlos isn't this a BC break? changing the visibility in a minor?

Reply:
I'd say no as it's `@internal`.

Reply:
yeah, it wasn't, it was tagged internal because making it `private` would be a BC break so it could be done on the next major, here it is just a plain BC break, will be reverted here https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5920 if someone cares

Reply:
@SpacePossum , it was `@internal` on v3.0.0, so we can change it to private at 3.x freely
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v3.0.0/src/Fixer/ClassNotation/ClassAttributesSeparationFixer.php#L43